src/Management/test/Endpoint.Test/Actuators/Health/EndpointMiddlewareTest.cs

```diff
@@ -313,10 +321,7 @@ public async Task GetStatusCode_OverrideUseStatusCodeFromResponseInConfiguration
         response.StatusCode.Should().Be(HttpStatusCode.OK);
 
         string json = await response.Content.ReadAsStringAsync();
-
-        json.Should().Contain("""
-            "status":"DOWN"
-            """);
+        json.Should().Be("{\"status\":\"DOWN\"}");
```

Comment:
I find the original assertion using a raw string easier to read, don't you agree?

Reply:
Code was changed from:
```c#
json.Should().Be("""
    "status":"DOWN"
    """);
```
to:
```c#
json.Should().Be("""{"status":"DOWN"}""");
```
I find the former easier to read, and easier to diff when pasting something over it.

Reply:
The original assertion was also `.Contain`. So the new version uses raw string for readability, and the full string so we don't need to separately assert the lack of other details

Reply:
Whether it uses `.Be` or `.Contains`, my point is about the line breaks.